<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>562</width>
       <height>512</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="uiGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="17" column="2">
        <widget class="QPushButton" name="uiEditKeyFileProperties">
         <property name="text">
          <string>Edit…</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Tab size:</string>
         </property>
         <property name="buddy">
          <cstring>uiTabSize</cstring>
         </property>
        </widget>
       </item>
       <item row="19" column="2">
        <widget class="QComboBox" name="uiKeyFileLength">
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_3">
         <property name="toolTip">
          <string>Path to libCURL executable</string>
         </property>
         <property name="text">
          <string>Libcurl path:</string>
         </property>
         <property name="buddy">
          <cstring>uiLibcurl</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="uiEnableFakeVim">
         <property name="text">
          <string>Enable FakeVim (restart required)</string>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="QPushButton" name="uiDefaultFileProperties">
         <property name="text">
          <string>Edit…</string>
         </property>
        </widget>
       </item>
       <item row="18" column="2">
        <widget class="QComboBox" name="uiS2KResultsPoolSize">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">128</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="uiDisplayLineNumbers">
         <property name="text">
          <string>Display line numbers</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_7">
         <property name="toolTip">
          <string>Libcurl command line parameters</string>
         </property>
         <property name="text">
          <string>Parameters:</string>
         </property>
         <property name="buddy">
          <cstring>uiLibcurlParameters</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="uiWindowsEol">
         <property name="toolTip">
          <string>&quot;Windows&quot; end of line: CR+LF</string>
         </property>
         <property name="text">
          <string>&quot;Windows&quot; end of line: CR+LF</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSpinBox" name="uiTabSize">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>40</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="13" column="2">
        <widget class="QPushButton" name="uiLibcurlBrowse">
         <property name="text">
          <string>Browse…</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="uiLastUsedDirectory">
         <property name="toolTip">
          <string>Retain last used folder between EncryptPad sessions</string>
         </property>
         <property name="text">
          <string>Save last used folder</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Default file properties:</string>
         </property>
         <property name="buddy">
          <cstring>uiDefaultFileProperties</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="uiFontFamily">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>[example: Monospace (Size: 12)]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="uiWordWrap">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Word wrap</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1" colspan="2">
        <widget class="QLineEdit" name="uiLibcurlParameters"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Recent files:</string>
         </property>
         <property name="buddy">
          <cstring>uiRecentFiles</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Font:</string>
         </property>
         <property name="buddy">
          <cstring>uiSelectFont</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="3">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="uiEnableBakFiles">
         <property name="text">
          <string>Create .bak files</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="uiLibcurl"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="uiSelectFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Font…</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <layout class="QHBoxLayout" name="layout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="uiRecentFiles">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="17" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Key file encryption properties:</string>
         </property>
         <property name="buddy">
          <cstring>uiEditKeyFileProperties</cstring>
         </property>
        </widget>
       </item>
       <item row="18" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="toolTip">
          <string>This controls how many times you can save without entering a passphrase</string>
         </property>
         <property name="text">
          <string>S2K derivations from a passphrase:</string>
         </property>
         <property name="buddy">
          <cstring>uiS2KResultsPoolSize</cstring>
         </property>
        </widget>
       </item>
       <item row="19" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Key file random sequence length (bytes):</string>
         </property>
         <property name="buddy">
          <cstring>uiKeyFileLength</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uiAppearance">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Icon theme (restart required):</string>
           </property>
           <property name="buddy">
            <cstring>uiIconTheme</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="uiIconTheme">
           <item>
            <property name="text">
             <string>Auto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dark</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>uiFontFamily</tabstop>
  <tabstop>uiSelectFont</tabstop>
  <tabstop>uiDisplayLineNumbers</tabstop>
  <tabstop>uiWordWrap</tabstop>
  <tabstop>uiWindowsEol</tabstop>
  <tabstop>uiTabSize</tabstop>
  <tabstop>uiEnableFakeVim</tabstop>
  <tabstop>uiRecentFiles</tabstop>
  <tabstop>uiLastUsedDirectory</tabstop>
  <tabstop>uiEnableBakFiles</tabstop>
  <tabstop>uiLibcurl</tabstop>
  <tabstop>uiLibcurlBrowse</tabstop>
  <tabstop>uiLibcurlParameters</tabstop>
  <tabstop>uiDefaultFileProperties</tabstop>
  <tabstop>uiEditKeyFileProperties</tabstop>
  <tabstop>uiS2KResultsPoolSize</tabstop>
  <tabstop>uiKeyFileLength</tabstop>
  <tabstop>uiIconTheme</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>695</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>695</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiSelectFont</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>selectFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiEditKeyFileProperties</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>editKeyFileProperties()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiDefaultFileProperties</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>editDefaultFileProperties()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFont()</slot>
  <slot>editDefaultFileProperties()</slot>
  <slot>editKeyFileProperties()</slot>
 </slots>
</ui>
